src/components/AddNewOrganizationModal.tsx

```diff
@@ -110,7 +110,7 @@ export default function AddNewOrganizationModal(props: AddNewOrganizationModalPr
         strings.ORGANIZATION_CREATED_MSG,
         strings.formatString(strings.ORGANIZATION_CREATED_TITLE, response.organization.name)
       );
-      reloadData(response.organization.id);
```

Comment:
Why are we reverting this change? I just tested adding a new org in Vercel and I'm not getting changed into that org

Reply:
we never changed to new org.. we only displayed a page message saying you can access your orgs in the list above.
only time we change org if user did not have an org to start with. that still works.
not sure why i assumed we changed orgs.. it was wrong.